atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/DbKvs.java

```diff
@@ -564,6 +566,11 @@ public void checkAndSet(CheckAndSetRequest checkAndSetRequest) throws CheckAndSe
         }
     }
 
+    @Override
+    public void multiCheckAndSet(MultiCheckAndSetRequest multiCheckAndSetRequest) throws MultiCheckAndSetException {
+        throw new UnsupportedOperationException("DbKvs does not support multi-checkAndSet operation!");
+    }
```

Comment:
Will raise a new PR to deal with this. This is complex enough to be done in a separate PR